### PR TITLE
translate documentation for instance method lifecycle

### PR DESCRIPTION
**Pull request by vendywira**

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -1782,99 +1782,99 @@ type: api
...
 
-- **Arguments:**
+- **Argumen:**
   - `{Function} [callback]`
 
-- **Usage:**
+- **Penggunaan:**
 
-  Defer the callback to be executed after the next DOM update cycle. Use it immediately after you've changed some data to wait for the DOM update. This is the same as the global `Vue.nextTick`, except that the callback's `this` context is automatically bound to the instance calling this method.
+  Menunda eksekusi dari *callback* setelah siklus pembaharuan DOM berikutnya. Gunakan method ini sesegera mungkin setelah anda melakukan perubahan data untuk menunggu pembaharuan *DOM*. Ini sama halnya dengan global `Vue.nextTick`, kecuali dalam kontek `this` pada *callback* yang akan otomatis tertuju kepada *instance* vue pemanggil.
```
Comment (mandaputtra) [inline src/v2/api/index.md]: konteks

**Review by mandaputtra — CHANGES_REQUESTED** [review]
LGTM

File: src/v2/api/index.md (hunk 1 above)
Comment (vendywira) [reply src/v2/api/index.md]: done

**Review by mandaputtra — APPROVED** [review]

**Comment by mandaputtra** [comment]
Okay thanks mas atas kontribusinya! 😃 💯

**Pull request merged**